app/src/main/res/layout/activity_game_activity.xml

```
@@ -193,7 +214,8 @@
                         android:layout_marginTop="5dp"
                         android:text="10(10)"
                         app:layout_constraintStart_toEndOf="@+id/greenPointsImageView3"
-                        app:layout_constraintTop_toBottomOf="@+id/Player1BluePointsTV" />
+                        app:layout_constraintTop_toBottomOf="@+id/Player1BluePointsTV"
+                        tools:ignore="NotSibling" />
```

I don't see any use for that line. It doesn't hide any errors.